charts/latest/csi-driver-nfs/templates/crd-csi-snapshot.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.externalSnapshotter.enabled -}}
+{{- if and .Values.externalSnapshotter.enabled .Values.externalSnapshotter.customResourceDefinitions.enabled -}}
```

Comment:
Cant comment on line 8 below, but need (annotation) added:
 "helm.sh/resource-policy": keep

Reply:
Yup. We can add it